bentoml/deployment/serverless.py

```diff
@@ -92,3 +98,28 @@ def deploy_with_serverless(platform, archive_path, extra_args):
     output_path = generate_serverless_bundle(bento_service, platform, archive_path, extra_args)
     subprocess.call(['serverless', 'deploy'], cwd=output_path)
     return output_path
+
+
+def delete_serverless_deployment(platform, service_name, additional_info):
+    yaml = YAML()
+    temp_dir = tempfile.mkdtemp()
+    saved_path = os.path.join(temp_dir, 'serverless.yml')
+    if platform == 'google-python':
+        config = {
+            "service": service_name,
+            "provider": {
+                "name": "google",
+                "region": additional_info.get('region', DEFAULT_GCP_REGION)
+            }
+        }
+    elif platform == 'aws-lambda' or platform == 'aws-lambda-py2':
+        config = {
+            "service": service_name,
+            "provider": {
+                "name": "aws",
+                "region": additional_info.get('region', DEFAULT_AWS_REGION)
+            }
+        }
+    yaml.dump(config, Path(saved_path))
+    subprocess.call(['serverless', 'remove'], cwd=temp_dir)
+    return
```

Comment:
would be great to have this function to return True when the deletion was successful and False when it failed to delete

Reply:
done